src/ui/triggers_main_area.ui

```diff
@@ -880,12 +925,7 @@
             </palette>
            </property>
            <property name="toolTip">
-            <string>The trigger will only fire if all conditions on the list 
-have been met within the specified line margin. 
-If this option is not set the trigger will fire if any 
-conditon on the list has been met. Read more 
-about the additional implications in the manual 
-as this is a complex topic.</string>
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The trigger will only fire if all conditions on the list have been met within the specified line delta, and captures will be saved in &lt;span style=&quot; font-style:italic;&quot;&gt;multimatches&lt;/span&gt; instead of &lt;span style=&quot; font-style:italic;&quot;&gt;matches&lt;/span&gt;. If this option is not set the trigger will fire if any conditon on the list has been met.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
😄 The HTML formatting is appreciated - I would note the point that using short HTML tags like:    `&lt;i&gt;`...`&lt;/i&gt;`    
{i,e, `<i>`...`</i>`}    
is easier on those who will have to read and translated the (source) text for Mudlet 4.0 compared to:    
`&lt;span style=&quot; font-style:italic;&quot;&gt;`...`&lt;/span&gt;`    
{i.e. `<span style=" font-style:italic;">`...`</span>`}    
but, of course the Qt Designer plugin does not make it easy to keep the "simple" form in place. \**sigh*\*